[2024-05-23T21:02:52.956+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-05-23T21:02:52.994+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: kaggle_dataset.transformacion manual__2024-05-23T01:15:45.195685+00:00 [queued]>
[2024-05-23T21:02:53.013+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: kaggle_dataset.transformacion manual__2024-05-23T01:15:45.195685+00:00 [queued]>
[2024-05-23T21:02:53.015+0000] {taskinstance.py:2306} INFO - Starting attempt 16 of 20
[2024-05-23T21:02:53.047+0000] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): transformacion> on 2024-05-23 01:15:45.195685+00:00
[2024-05-23T21:02:53.058+0000] {warnings.py:110} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61: DeprecationWarning: This process (pid=840) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-05-23T21:02:53.060+0000] {standard_task_runner.py:63} INFO - Started process 864 to run task
[2024-05-23T21:02:53.061+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'kaggle_dataset', 'transformacion', 'manual__2024-05-23T01:15:45.195685+00:00', '--job-id', '31', '--raw', '--subdir', 'DAGS_FOLDER/kaggle_dataset.py', '--cfg-path', '/tmp/tmpq5va39z_']
[2024-05-23T21:02:53.065+0000] {standard_task_runner.py:91} INFO - Job 31: Subtask transformacion
[2024-05-23T21:02:53.135+0000] {task_command.py:426} INFO - Running <TaskInstance: kaggle_dataset.transformacion manual__2024-05-23T01:15:45.195685+00:00 [running]> on host 7ef43f6735eb
[2024-05-23T21:02:53.278+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='kaggle_dataset' AIRFLOW_CTX_TASK_ID='transformacion' AIRFLOW_CTX_EXECUTION_DATE='2024-05-23T01:15:45.195685+00:00' AIRFLOW_CTX_TRY_NUMBER='16' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-05-23T01:15:45.195685+00:00'
[2024-05-23T21:02:53.280+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-05-23T21:02:53.618+0000] {logging_mixin.py:188} INFO -        country_id  ... gross_domestic_product_(gdp)
0               4  ...                 1.731436e+09
1               4  ...                 1.812838e+09
2               4  ...                 1.647900e+09
3               4  ...                 1.702716e+09
4               4  ...                 2.061729e+09
...           ...  ...                          ...
10507         894  ...                 2.586816e+10
10508         894  ...                 2.631198e+10
10509         894  ...                 2.331036e+10
10510         894  ...                 1.811064e+10
10511         894  ...                 2.131337e+10

[10512 rows x 26 columns]
[2024-05-23T21:02:53.621+0000] {logging_mixin.py:188} INFO - Index(['country_id', 'country', 'year', 'ama_exchange_rate',
       'imf_based_exchange_rate', 'population', 'currency', 'per_capita_gni',
       'agriculture,hunting,forestry,fishing(isic_a_b)',
       'changes_in_inventories', 'construction_(isic_f)',
       'exports_of_goods_and_services', 'final_consumption_expenditure',
       'general_government_final_consumption_expenditure',
       'gross_capital_formation', 'gross_fixed_capital_formation',
       'household_consumption_expenditure', 'imports_of_goods_and_services',
       'manufacturing_(isic_d)', 'mining,_manufacturing,_utilities(isic_c_e)',
       'other_activities_(isic_j_p)', 'total_value_added',
       'transport,storage_and_communication(isic_i)',
       'wholesale,retail_trade,restaurants_and_hotels(isic_g_h)',
       'gross_national_income(gni)_in_usd', 'gross_domestic_product_(gdp)'],
      dtype='object')
[2024-05-23T21:02:53.625+0000] {logging_mixin.py:188} INFO - country_id                                                   int64
country                                                     object
year                                                         int64
ama_exchange_rate                                          float64
imf_based_exchange_rate                                    float64
population                                                   int64
currency                                                    object
per_capita_gni                                               int64
agriculture,hunting,forestry,fishing(isic_a_b)             float64
changes_in_inventories                                     float64
construction_(isic_f)                                      float64
exports_of_goods_and_services                              float64
final_consumption_expenditure                              float64
general_government_final_consumption_expenditure           float64
gross_capital_formation                                    float64
gross_fixed_capital_formation                              float64
household_consumption_expenditure                          float64
imports_of_goods_and_services                              float64
manufacturing_(isic_d)                                     float64
mining,_manufacturing,_utilities(isic_c_e)                 float64
other_activities_(isic_j_p)                                float64
total_value_added                                          float64
transport,storage_and_communication(isic_i)                float64
wholesale,retail_trade,restaurants_and_hotels(isic_g_h)    float64
gross_national_income(gni)_in_usd                          float64
gross_domestic_product_(gdp)                               float64
dtype: object
[2024-05-23T21:02:53.702+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-05-23T21:02:53.704+0000] {taskinstance.py:2905} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/taskinstance.py", line 465, in _execute_task
    result = _execute_callable(context=context, **execute_callable_kwargs)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/taskinstance.py", line 432, in _execute_callable
    return execute_callable(context=context, **execute_callable_kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/baseoperator.py", line 400, in wrapper
    return func(self, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/operators/python.py", line 235, in execute
    return_value = self.execute_callable()
                   ^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/operators/python.py", line 252, in execute_callable
    return self.python_callable(*self.op_args, **self.op_kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/airflow/dags/kaggle_dataset.py", line 65, in transform_task
    load_df_to_sql(df, 'etl_kaggle', 'tmp_global_economic_ind', sql_connid_postgrest)
  File "/opt/airflow/dags/utils.py", line 33, in load_df_to_sql
    for item in row
                ^^^
NameError: name 'row' is not defined. Did you mean: 'rows'?
[2024-05-23T21:02:53.737+0000] {taskinstance.py:1206} INFO - Marking task as UP_FOR_RETRY. dag_id=kaggle_dataset, task_id=transformacion, run_id=manual__2024-05-23T01:15:45.195685+00:00, execution_date=20240523T011545, start_date=20240523T210252, end_date=20240523T210253
[2024-05-23T21:02:53.762+0000] {standard_task_runner.py:110} ERROR - Failed to execute job 31 for task transformacion (name 'row' is not defined; 864)
[2024-05-23T21:02:53.802+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 1
[2024-05-23T21:02:53.818+0000] {warnings.py:110} WARNING - /home/***/.local/lib/python3.12/site-packages/***/models/baseoperator.py:1296: AirflowProviderDeprecationWarning: Call to deprecated class PostgresOperator. (Please use `***.providers.common.sql.operators.sql.SQLExecuteQueryOperator`.Also, you can provide `hook_params={'schema': <database>}`.)
  result = cls.__new__(cls)

[2024-05-23T21:02:53.841+0000] {taskinstance.py:3498} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-05-23T21:02:53.845+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
